packages/client/src/elements/AppBar/index.tsx

```diff
@@ -1,62 +1,134 @@
-import React, { MouseEvent, useState } from 'react';
+import React, { MouseEvent, useEffect, useMemo, useState } from 'react';
+import { Redirect, Route, Switch, useHistory } from 'react-router-dom';
 
-import { useMediaDevice } from '../../hooks';
+import { routes } from '../../constants';
+import { Dashboard } from '../../containers';
+import { useCurrentUser, useMediaDevice } from '../../hooks';
 import { AdminIcon, DashboardIcon, HistoryIcon, SettingsIcon } from '../../icons';
 import TabbedLayout from '../layouts/TabbedLayout';
-import { TabProps } from '../layouts/TabbedLayout/Tab/Tab.types';
-import { TabsWrapper, Wrapper } from './AppBar.styled';
+import { hasAccess } from '../layouts/TabbedLayout/Tabs/Tabs.utils';
+import { TabPageWrapper, TabsWrapper, Wrapper } from './AppBar.styled';
+import { AppBarItem } from './AppBar.types';
+
+const tabs: Array<AppBarItem> = [
+  {
+    index: 0,
+    indexMobile: 1,
+    path: '/',
+    exact: true,
+    label: 'menu.dashboard',
+    icon: <DashboardIcon />,
+    component: <Dashboard />,
+  },
+  {
+    index: 1,
+    indexMobile: 0,
+    path: '/history',
+    label: 'menu.history',
+    icon: <HistoryIcon />,
+    component: <p>history</p>,
+  },
+  {
+    index: 2,
+    path: '/admin',
+    label: 'menu.admin',
+    icon: <AdminIcon />,
+    onlyAdmin: false,
+    component: <p>admin</p>,
+  },
+  {
+    index: 3,
+    path: '/settings',
+    label: 'menu.settings',
+    icon: <SettingsIcon />,
+    component: <p>settings</p>,
+  },
+];
 
 const AppBar = () => {
-  const [activeTab, setActiveTab] = useState<number>(1);
   const { isMobile } = useMediaDevice();
+  const [currentUser] = useCurrentUser();
+  const sortedItems = useMemo(
+    () =>
+      tabs
+        .filter((tab) => hasAccess(tab.onlyAdmin, currentUser))
+        .map((tab) => {
+          const { index, indexMobile } = tab;
+          const reindexTab = tab;
+          if (isMobile) {
+            reindexTab.index = indexMobile !== undefined ? indexMobile : index;
```

Comment:
```suggestion
            reindexTab.index = indexMobile ?? index;
```
i think it's better way